fineract-provider/src/main/java/org/apache/fineract/notification/service/NotificationDomainServiceImpl.java

```diff
@@ -345,4 +347,19 @@ private Set<Long> getNotifiableUserIds(Long officeId, String permission) {
         Collection<AppUser> usersWithPermission = users.stream().filter(aU -> aU.hasAnyPermission(permission, "ALL_FUNCTIONS")).toList();
         return usersWithPermission.stream().map(AppUser::getId).collect(toSet());
     }
+
+    private static class LoanStatusChangedListener implements BusinessEventListener<LoanStatusChangedBusinessEvent> {
```

Comment:
Please move this logic into LoanWritePlatformServiceJpaRepositoryImpl or create a new service which will take care of the logic. I reckon the NotificationDomainService is not the proper place!

Reply:
Done!

Reply:
Done It was moved